src/dlgIRC.cpp

```diff
@@ -173,7 +173,7 @@ QPair<bool, QString> dlgIRC::sendMsg(const QString& target, const QString& messa
         mPingStarted = QDateTime::currentMSecsSinceEpoch();
     }
 
-    bool rv = connection->sendCommand(command);
+    connection->sendCommand(command);
```

Comment:
If this call genuinely won't return a `false` for us then I guess this is reasonable.  When could it have done BTW - if we used the library for something else that we do not currently do perhaps?